<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ModPersonInfoDlg</class>
 <widget class="QDialog" name="ModPersonInfoDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>513</width>
    <height>416</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>修改人员信息</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#ModPersonInfoDlg 
{
	background:  rgba(32, 80, 96, 100);
	border-image: url(:/images/images/userinput_bg.png);
}

#mainForm 
{ 
background-color:rgba(0, 0, 0, 80); 
}

QLabel { 
color: white; 
min-height: 30px;
font-size: 20px;
}
QLineEdit { 
color: balck; 
min-width: 200px;
min-height: 30px;
font-size: 20px;
}
QSpinBox { 
color: balck; 
min-width: 200px;
min-height: 30px;
font-size: 20px;
}
QDateEdit { 
color: balck; 
min-width: 200px;
min-height: 30px;
font-size: 20px;
}

QPushButton{
color : white;
background-color: #27a9e3;
border-width:0px;
border-radius:10px;
border-style: outset;
min-height: 40px;
font-size: 30px;
}

QPushButton:pressed{
background-color: #66c011;
padding-left:5px;
padding-top:5px;
}
#infoLbl
{ font-size: 40px; min-height: 50px;}</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <spacer name="verticalSpacer_2">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Fixed</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>10</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QLabel" name="infoLbl">
     <property name="text">
      <string>修改人员信息</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer_3">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>26</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QWidget" name="mainForm" native="true">
       <layout class="QVBoxLayout" name="verticalLayout">
        <property name="leftMargin">
         <number>35</number>
        </property>
        <property name="topMargin">
         <number>35</number>
        </property>
        <property name="rightMargin">
         <number>35</number>
        </property>
        <property name="bottomMargin">
         <number>35</number>
        </property>
        <item>
         <layout class="QFormLayout" name="formLayout">
          <item row="0" column="0">
           <widget class="QLabel" name="Label">
            <property name="text">
             <string>姓名：</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QLineEdit" name="usernameEdit"/>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="Label_2">
            <property name="text">
             <string>年龄：</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QSpinBox" name="ageEdit"/>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="Label_3">
            <property name="text">
             <string>电话：</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QLineEdit" name="telephoneEdit"/>
          </item>
          <item row="3" column="0">
           <widget class="QLabel" name="Label_4">
            <property name="text">
             <string>卡片：</string>
            </property>
           </widget>
          </item>
          <item row="3" column="1">
           <widget class="QLineEdit" name="cardidEdit">
            <property name="readOnly">
             <bool>true</bool>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QPushButton" name="cancel">
            <property name="text">
             <string>取消</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="ok">
            <property name="text">
             <string>确定</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>27</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>usernameEdit</tabstop>
  <tabstop>ageEdit</tabstop>
  <tabstop>telephoneEdit</tabstop>
  <tabstop>cardidEdit</tabstop>
  <tabstop>ok</tabstop>
  <tabstop>cancel</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
